1/index.js

```diff
@@ -1 +1,11 @@
-console.log('1!');
+//lesson01--------------------------------------------------
+//chatGPT prompt："javascript DOM作成 li テキスト「これです」"
+//reference:https://www.wakuwakubank.com/posts/306-javascript-dom/#index_id4
+//reference:https://stackoverflow.com/questions/50250214/how-to-use-insertbefore-without-second-parameter
+
+//create list item
+const newList = document.getElementById('newlist'); // get:ID='newlist'
+console.log(newList);
+const listItem = document.createElement('li'); // create<li></li>
+const newContent =  listItem.textContent = 'これです' // add:text -> これです
```

Comment:
IMO
I didn't think it was necessary to store it in a variable.
```suggestion
listItem.textContent = 'これです' // add:text -> これです
```

Reply:
Thank you!
I agree....!
Why did I store it variable...

Reply:
https://github.com/megusasuke0/frontendhandson/commit/d1cd14bfa81aaa8d39ce0a8fd4d90cb7be53f627

Reply:
you didn't fix your code like below.
listItem.textContent = 'これです' ;

Reply:
Oh! Sorry...Thank you for your notice!!
I've done it!

https://github.com/megusasuke0/frontendhandson/commit/9a509e14a009792163e824cbd781adeaa36e2da9